shared/common-adapters/dumb.native.js

```diff
@@ -22,7 +22,8 @@ const tabBarButtonMap: DumbComponentMap<TabBarButton> = {
 const tabBarBaseMock = {
   style: {flex: 1},
   children: [
-    (<TabBarItem label='One' selected onClick={() => {}}>
+    // eslint-disable-next-line react/jsx-boolean-value
+    (<TabBarItem label='One' selected={true} onClick={() => {}}>
```

Comment:
New flow doesn't like default true JSX attributes. This error was being hidden because the flow types weren't actually being enforced.